stories/s2l1c.stories.js

```diff
@@ -124,13 +124,14 @@ export const GetMapProcessing = () => {
 };
 
 export const GetMapWMSMaxCC20vs60 = () => {
-  const layerS2L1C20 = new S2L1CLayer(instanceId, s2l1cLayerId, null, null, null, null, null, 20);
-  const layerS2L1C60 = new S2L1CLayer(instanceId, s2l1cLayerId, null, null, null, null, null, 100);
+  const layerS2L1C20 = new S2L1CLayer({ instanceId, layerId, maxCloudCoverPercent: 20 });
+  const layerS2L1C60 = new S2L1CLayer({ instanceId, layerId, maxCloudCoverPercent: 100 });
```

Comment:
Sorry, that is my mistake from one of the previous PRs. `maxCloudCoverPercent` of the second one should be 60 or the name(s) should be fixed. Can you maybe fix it here?

Reply:
Sure, though it still doesn't work (wrong dates - 2014). But that should probably be solved in another PR.

Reply:
Thanks! I know, but just to put the `maxCloudCoverPercent` out of the mind. I agree about the dates.